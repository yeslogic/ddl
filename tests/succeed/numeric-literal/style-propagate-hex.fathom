//~ test-normalisation = true

u8_add 0x2 1
