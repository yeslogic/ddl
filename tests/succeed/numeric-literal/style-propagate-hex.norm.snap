stdout = '''
0x3 : U8
'''
stderr = ''
